src/test/kotlin/racingcar/CarTest.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+
+class CarTest : FunSpec({
+    context("자동차에 4 이상의 힘을 가하면 전진한다") {
+        withData(listOf(4, 5, 6, 7, 8, 9)) { power ->
+            val car = Car(engine = { power })
+            car.move()
+            car.location shouldBe 1
+        }
+    }
+    context("자동차에 4 미만의 힘을 가하면 움직이지 않는다") {
+        withData(listOf(0, 1, 2, 3)) { power ->
+            val car = Car(engine = { power })
+            car.move()
+            car.location shouldBe 0
+        }
+    }
```

Comment:
Parameterized Test를 잘 활용하면 아래와 같이 하나의 테스트 케이스로 변경해볼 수 있을 것 같아요. 😉
```suggestion
    context("자동차는 힘에 따라서 전진 혹은 정지한다") {
        withData(
            row(0, 0),
            row(1, 0),
            row(2, 0),
            row(3, 0),
            row(4, 1),
            row(5, 1),
            row(6, 1),
            row(7, 1),
            row(8, 1),
            row(9, 1),
        ) { (power, expected) ->
            val car = Car(engine = { power })
            car.move()
            car.location shouldBe expected
        }
    }
```

Reply:
테스트를 분리하는 것과 통합하는 것은 어떤 차이가 있을까요? 취향 차이일까요?